blackjack/api/views.py

```diff
@@ -8,7 +9,15 @@
 from rest_framework.generics import ListAPIView, RetrieveAPIView, get_object_or_404
 
 
+class Game:
+    deck_cards = [2, 2, 2, 2, 3, 3, 3, 3, 4, 4, 4, 4, 5, 5, 5, 5, 6, 6, 6, 6, 7, 7, 7, 7, 8, 8, 8, 8, 9, 9, 9, 9,
+                  10, 10, 10, 10, 'J', 'J', 'J', 'J', 'Q', 'Q', 'Q', 'Q', 'K', 'K', 'K', 'K', 'A', 'A', 'A', 'A'
+                  ]
 
-@api_view(["GET"])
-def deal(request):
-    return JsonResponse(data={'key':'value'}, status=status.HTTP_200_OK)
\ No newline at end of file
+    def __init__(self):
+        random.shuffle(self.deck_cards)
```

Comment:
🎆 your python skills are on point 🎆

Reply:
full disclosure, I did steal this from our previous python code of the game so don't get too excited